website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -19,9 +19,9 @@ dbt commands are run against dbt Cloud's infrastructure and benefit from:
 <Lightbox src="/img/docs/dbt-cloud/cloud-cli-overview.jpg" title="Diagram of how the dbt Cloud CLI works with dbt Cloud's infrastructure to run dbt commands from your local command line." />
 
 ## Prerequisites 
-The dbt Cloud CLI is available in all [deployment regions](/docs/cloud/about-cloud/access-regions-ip-addresses) and for both multi-tenant and single-tenant accounts (Azure single-tenant not supported at this time).
+The dbt Cloud CLI is available in all [deployment regions](/docs/cloud/about-cloud/access-regions-ip-addresses) and for both multi-tenant and single-tenant accounts.
```

Comment:
I think the preferred nomenclature is Virtual Private Cloud but maybe that's a conversation for a different time

Reply:
thanks @matthewshaver ! happy to update this but this page here: https://docs.getdbt.com/docs/cloud/about-cloud/tenancy also has single tenant? i thought vpc was a tier or something?